File: eCinema.Model/Glumac.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace eCinema.Model
{
    public class Glumac
    {
        public int GlumacId { get; set; }
        public string Ime { get; set; }
        public string Prezime { get; set; }
        public string? Slika { get; set; }
        public string ImePrezime { get; set; }


        public override string ToString()
        {
            return ImePrezime;
        }
    }
}

File: ecinema_mobile/lib/models/glumac.dart
import 'package:json_annotation/json_annotation.dart';

part 'glumac.g.dart';

@JsonSerializable()
class Glumac {
  int glumacId;
  String ime;
  String prezime;
  String? slika;
  String imePrezime;

  Glumac(
      {required this.glumacId,
      required this.ime,
      required this.prezime,
      this.slika,
      required this.imePrezime});

  factory Glumac.fromJson(Map<String, dynamic> json) => _$GlumacFromJson(json);
  Map<String, dynamic> toJson() => _$GlumacToJson(this);
}
